src/components/navigation/FlashNotification.tsx

```diff
@@ -24,14 +24,19 @@ export function FlashNotification(props: Props) {
 
   return (
     <AirshipDropdown bridge={bridge} backgroundColor={theme.modal} onPress={onPress}>
-      <AntDesignIcon name="checkcircle" size={theme.rem(2)} style={styles.icon} />
-      <Text style={styles.text}>{message}</Text>
-      <ModalFooter onPress={handleClose} />
+      <View style={styles.container}>
+        <AntDesignIcon name="checkcircle" size={theme.rem(2)} style={styles.icon} />
+        <Text style={styles.text}>{message}</Text>
+        <ModalFooter onPress={handleClose} />
```

Comment:
This is not a modal, so using the `ModalFooter` in here is incorrect. However, as this is a P0 blocking a release, we can merge this and fix it properly later.